libs/langchain/langchain/chains/graph_qa/anzograph.py

```diff
@@ -0,0 +1,152 @@
+"""
```

Comment:
this should be in community as all the other graph chains now are

Reply:
Hi, 
Not really, please reconsider, that is exactly the folder where all the similar chains are. I based AnzoGraphQAChain on sparql.py and ontotext_graphdb.py modules, which are in this very same directory (langchain/chains/graph_qa).
Within langchain_community we just have the graphs modules (langchain_community/graphs) and all the providers did the same way.